docker/v1.24.0/ubuntu18.04/Dockerfile

```diff
@@ -0,0 +1,134 @@
+#
+#  Copyright (c) 2017-2024, Intel Corporation
+#
+#  SPDX-License-Identifier: BSD-3-Clause
+
+ARG LLVM_VERSION=17.0
+
+FROM ubuntu:18.04 AS llvm_build_step1
+LABEL maintainer="Dmitry Babokin <dmitry.y.babokin@intel.com>"
+SHELL ["/bin/bash", "-c"]
+
+ARG REPO=ispc/ispc
+ARG SHA=main
```

Comment:
I noticed that starting v1.22.0 we forgot updating SHA in Dockerfiles. Could you update it here and for Dockerfiles for previous releases?

Reply:
Done